content/docs/command-reference/add.md

```diff
@@ -30,7 +30,7 @@ The `targets` are the files or [directories](#adding-entire-directories) to add.
 They get stored in the <abbr>cache</abbr> by default (use the `--no-commit`
 option to avoid this, and `dvc commit` to finish the process when needed).
 
-> See also `dvc.yaml` and `dvc run` for more advanced ways to track and version
+> See also `dvc.yaml` and `dvc n` for more advanced ways to track and version
```

Comment:
```suggestion
> See also `dvc.yaml` and `dvc stage add` for more advanced ways to track and version
```

Reply:
I can no longer commit suggestions. Otherwise I'd apply this, approve and merge the PR.

Reply:
Interesting. Probably a GH bug or something. I have that button as of now. 

![image](https://user-images.githubusercontent.com/476310/142184877-cd9f1204-8986-45eb-afed-fc111bb10648.png)

Reply:
Yeah got it again now 🤷